lci_strategic_plugin/CMakeLists.txt

```diff
@@ -93,10 +93,10 @@ install(DIRECTORY launch config
 
 catkin_add_gmock(${PROJECT_NAME}-test 
  test/test_main.cpp
- #test/test_fixture.h
- #test/test_strategic_plugin_helpers.cpp
+ test/test_fixture.h
+ ##test/test_strategic_plugin_helpers.cpp
```

Comment:
This file was modified but is commented out in the CMake why?

Reply:
That file is edited just so that it builds, but unit tests fail so I kept it turned off because it was turned off anyways and as an indication to get back to it in the future. All test added yesterday for handle case failure passes.